lib/ClangImporter/ImporterImpl.h

```diff
@@ -971,13 +1012,11 @@ class LLVM_LIBRARY_VISIBILITY ClangImporter::Implementation
   ///
   /// \returns the imported function type, or null if the type cannot be
   /// imported.
-  Type importFunctionType(DeclContext *dc,
-                          const clang::FunctionDecl *clangDecl,
-                          ArrayRef<const clang::ParmVarDecl *> params,
-                          bool isVariadic,
-                          bool isFromSystemModule,
-                          DeclName name,
-                          ParameterList *&parameterList);
+  std::pair<Type, bool>
+  importFunctionType(DeclContext *dc, const clang::FunctionDecl *clangDecl,
```

Comment:
*These* probably need comment adjustments more than the other one, since the flag refers to the result rather than the type as a whole.